src/status_test.rs

```diff
@@ -11,9 +11,21 @@ use primitives::{
 };
 
 use httptest::{mappers::*, responders::*, Expectation, Server, ServerPool};
+use lazy_static::lazy_static;
 
 static SERVER_POOL: ServerPool = ServerPool::new(4);
 
+lazy_static! {
```

Comment:
I was thinking of something simpler, as we don't need everting to be static for every test case.
We can leave only a `RECENT` (3 mins) here and we can use it everywhere else to setup the test cases. `Duration::zero()` is as simple as it can get so I don't think we need to create a whole static for it.
IMO how `is_offline` had the `old_recency` set only for one test (when you need it and you're testing it) was clear enough, but I would like to hear your thoughts.

The setting of the correct times can be left as it was as well, we don't need statics for those as it also depends on the test and in the case of DateTime it is also time sensitive. I.e. if you are not careful, random tests might start failing, because the `THREE_MINUTES_AGO` was evaluated 4 minutes a go and now one of the tests fails because that's not a recent DateTime when it started execution.

Reply:
Yes, maybe exporting every single duration was too much. I only kept the two most used as global for all tests.